### PR TITLE
- add some german translations and correct one other

### PR DESCRIPTION
**Pull request by st-pa-fo**
- rename (only the user-facing part of) all buildings '%SPECIES% Colony' to 'Colony of %SPECIES%'
--> bunches them together in the buildingtype pedia page instead of smearing all over the list when in default alphabetic sorting order
- umbenennung (nur der nutzersichtbaren seite) aller gebäude '%SPEZIES%-Kolonie' zu 'Kolonie der %SPEZIES%'
--> so stehen sie in der standardmäßig alphabetisch sortierten pedia schön sauber beisammen statt über die ganze liste verstreut

### CODE REVIEW

**Comment by geoffthemedio** [comment]
@Vezzra any comment?

**Comment by Vezzra** [comment]
I'll try to take a look coming weekend.

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: I think "Fortsetzen" would be better

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:6530]: I don't think changing it like this is an improvement in this particular context. Sure, when saying something like "Sie näherten sich der Kolonie der Abaddoni", then this sounds better than "Sie näherten sich der Abaddoni-Kolonie". But as a label for a building "Kolonie der Abaddoni" sounds more awkward than "Abaddoni-Kolonie".

Same goes for all the other colony buildings of course.

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: Capital first letter for nouns: "Effekte", not "effekte"

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:8137]: Better: "Tabs für Einrückung verwenden"

Don't address the player/user in the second person. (If that's done in the original English string, it needs to be corrected accordingly there too, but not in this patch.)

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: While you are at it, the translation here could be improved too: "Durchschnittliche" instead of "Mittelmäßige".

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: Same here

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: And here.

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:8191]: Hm, maybe the translation could be improved here too, at least a bit. I think "Optimale" would be a bit better than "Perfekte" here...

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:8203]: Same here

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: Again: "Mittelmäßige" -> "Durchschnittliche"

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:8218]: "Perfekte" -> "Optimale"

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:8224]: Here too "Durchschnittliche" would be better than "Normale" I think...

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt:8230]: "besiedelten Asteroidengürteln" is probably a bit better than "besetzten Asteroidengürteln". "Besetzt" sounds too much like occupied in the military sense.

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: Ugh, that's getting difficult here. The correct/best wording in German for this actually should be (IMO): "...stark reduziert in Systemen mit blauen Sternen, weniger stark in Systemen mit weißen Sternen".

However, if the `[[STAR_BLUE]]` and `[[STAR_WHITE]]` should/must be used, using "...Sterne der Farbe..." probably can't be avoided, as these macros get substituted by "Blau" and "Weiß" I guess?

The question is, is it necessary to use the macros here? It certainly makes the translation more awkward. @geoffthemedio?

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: Better: "Telepathische Wahrnehmung: kann nahe besiedelte Welten wahrnehmen"

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: "Vorausahnende Wahrnehmung: kann Planeten wahrnehmen, die per Sternenstraßen verbunden sind".

Actually I'm wondering if there isn't a better term in German for "starlane" than "Sternenstraße", but I can't think of anything...

File: default/stringtables/de.txt
Comment (Vezzra) [inline default/stringtables/de.txt]: "Gemeinsame Wahrnehmung: Sichtbarkeit wird innerhalb derselben Spezies geteilt, unabhängig von Reichszugehörigkeit." The last part after the comma isn't actually there in the English string (maybe should?), but in German it's probably a good idea to add this.

That aside, "Sichtbarkeit wird ... geteilt" isn't really proper German, but I too can't think of a better way to say this.

**Review by Vezzra — CHANGES_REQUESTED** [review]
Ok, left a couple of comments/suggestions for improvements.

File: default/stringtables/de.txt
Comment (geoffthemedio) [reply default/stringtables/de.txt:6530]: the motivation was:

> bunches them together in the buildingtype pedia

File: default/stringtables/de.txt
Comment (geoffthemedio) [reply default/stringtables/de.txt]: Translations don't need to adhere rigidly to english macro use.

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: yeah that one was bugging me too .. but having seen a german youtuber talking about how he play fo for the first time, i think it works .. and t.b.h. havent played much of these games in my native tongue myself since englisch ist so ubiquitös ...

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt:8230]: was what i went for, outposts count, after all

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt:8137]: that is addressed to the translators ... so to you and me i guess

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: this is inside a comment to us translators. not visible to players

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt:6530]: ja also nochmal kurz auf deutsch, das mag nich viel besser oder schlechter für die einzelne kolonie sein, ob sie nun menschenkolonie heißt oder kolonie der menschen, aber in der pedia .. haben wir dann alle schön zusammen beim buchstaben k wie kolonie hintereinanderweg und dann alphabetisch nach spezies, anstatt wie bisher zuerst der name der spezies, was die koloniegebäudenamen derzeit über die komplette liste verstreut und sie ziemlich unschön zu benutzen macht. das soll das beheben.

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: wenn es wenigstens ein eszett gewesen wäre … hat das ein schweizer geschrieben? war mir gar nicht aufgefallen. ja sicherlich man kann noch einiges verbessern ..

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: yep and all that hyperactive hyperlinking can get a little too hyped at times i agree, in this case plain german without the hyper hyper scoots better i'd say as well

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: ✓

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: soo .. maybe let the not-quite-perfect-but-understandable german stand instead of no translation of that segment? you see in professional work life i do computer related translation stuff professionally .. there i hate to fall short of perfection as well. but in this case i'd say, if the 100% is not achievable, then the 75% is better than zero .. maybe?

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: agreed

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: motivation was that i wanted to keep it short for the game start screen when selecting species to play where we have only the little tooltip popover with the same text that has to go into the pedia page as well, but on that small species selection tooltip popover there some long text lines that are wrapped in a not so nice to look at manner .. anyway i'll take your suggestion verbatim.

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: taking that verbatim as well

File: default/stringtables/de.txt
Comment (st-pa-fo) [reply default/stringtables/de.txt]: > in the English string (maybe should?)

yes indeed, i got that over in another pr with the english minor tweakage stuff, it says their now smth like 'irregardless of affiliation'

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt:6530]: Ah, verstehe. Ok, das klingt natürlich vernünftig... I'll continue in English for the benefit of our Non-German speaking friends. 😉 

Ok, sounds reasonable - however, if this is really that much of an issue, equivalent changes should probably also be made in the other language stringtables. @geoffthemedio?

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt]: True. Not really an issue, but I prefer to have proper spelling even in comments. Maybe I'm too much of a perfectionist in this regard - I'll leave it to you if you want to correct this or not... 😉

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt]: Oh, stimmt - siehst, das ist mir auch gar nicht mal aufgefallen! Andererseits bin ich alte Rechtschreibung gewöhnt (also wie man sie noch vor 40 Jahren gelernt hatte), für mich schaut seit diesen Rechtschreibreformen der letzten Jahrzehnte vieles einfach falsch aus, da fällt mir dann sowas nimmer auf... :zany_face::zany_face::zany_face:

Wie auch immer, hast recht, das kann man auch in einem extra PR mal machen.

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt:8137]: True - same as above: not really an issue, it's just that I try to "follow the rules", or keep to a certain style even with comments. I'll leave it to you. 😃

File: default/stringtables/de.txt
Comment (geoffthemedio) [reply default/stringtables/de.txt:6530]: it's a reasonable change... I don't have a strong preference, though.

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt:8230]: I don't think I understand - what do you mean by "outposts count"...?

Not that it's really an issue. If you think "besetzt" works better, that's ok with me. Personally I just think "besiedelt" would be better.

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt:6530]: Ok, fair enough. Applying equivalent changes to the other stringtables should be done in separate PRs anyway, so not relevent here.

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt]: > and t.b.h. havent played much of these games in my native tongue myself

Yep, I know what you mean. With games I usually also usually use English, German translation tend to have issues (too long texts, awkward, sometimes even wrong/misleading translations, etc.). English just works better for me... :wink:

File: default/stringtables/de.txt
Comment (Vezzra) [reply default/stringtables/de.txt]: > but in this case i'd say, if the 100% is not achievable, then the 75% is better than zero

Yes, definitely. My statement was only meant as a comment, not an objection. As long as we can't come up with something better, we'll work with what we _can_ come up with...

**Review by Vezzra — APPROVED** [review]

**Comment by Vezzra** [comment]
@st-pa-fo, do you want to make some last changes? Otherwise this can be merged.

**Comment by st-pa-fo** [comment]
> @st-pa-fo, do you want to make some last changes? Otherwise this can be merged.

no more changes here. i'll try to make translations of the new species texts from english to some other languages as well but i would like to wait with that until the english versions are approved first. so, merge ahead ^^ and thanx

**Pull request merged**
